tools/generate-countries.ipynb

```diff
@@ -290,12 +311,37 @@
     "\n",
     "    modify_template(templatename, os.path.join(\"ipynb\", output_file_name), mappings, wwwroot)\n",
     "    assert os.path.exists(output_file_path), f\"{output_file_path} does not exist\"\n",
-    "    return output_file_name\n"
+    "    return output_file_name\n",
+    "\n",
+    "def create_ipynb_for_hungary(region, subregion='', templatename='', wwwroot=''):\n",
```

Comment:
```suggestion
    "def create_ipynb_for_hungary(region, subregion, templatename, wwwroot):\n",
```

Are the defaults needed? Setting subregion, templatename, and wwwroot to an empty string if not provided could cause some errors (like failing due to the template not existing) or problems (like just dumping all the html and ipynb files into your current directory)

Reply:
They are not needed. 
As for the 'subregion' it will be removed if that is okay.